django_cloud_deploy/skeleton/source_generator.py

```diff
@@ -30,41 +30,31 @@
 class _FileGenerator(object):  # pytype: disable=ignored-abstractmethod
```

Comment:
should this look like:

class Prompt(abc.ABC):
    @abc.abstractmethod
    def foo():

Reply:
Done.

Reply:
I removed @abstractmethd decorator for those methods. This is because those two methods can have variant parameters in subclasses.